app/features/send/send-confirmation.tsx

```diff
@@ -0,0 +1,404 @@
+    },
+    onError: (error) => {
+      console.error('Error getting quotes', error);
+    },
+    onSuccess: (data) => {
+      console.log('quotes', data);
+    },
+  });
+
+  useEffect(() => {
+    mutate();
+  }, [mutate]);
+
+  const handleConfirm = async () => {
+    let proofs: Proof[];
+    if (needsCrossMintSwap) {
+      if (!data?.quotes) {
+        throw new Error('Trying to swap, but cross mint quotes are not loaded');
+      }
+      // we will need amount + fee in proofs to pay the mint quote
+      const _proofAmtRequired = toMoney(
+        data.quotes.meltQuote.amount + data.quotes.meltQuote.fee_reserve,
+        account.currency,
+      );
+
+      const proofsToSwap: Proof[] = []; // TODO: create proofs from account for proofAmtRequired
+
+      // swap the proofs to the mint in the payment request
+      const { newProofs, change: _ } = await crossMintSwap(
+        new CashuWallet(new CashuMint(account.mintUrl)),
+        new CashuWallet(new CashuMint(sendToMintUrl)),
+        proofsToSwap,
+        data.quotes,
+      );
+      // TODO: handle change (add to db)
+      proofs = newProofs;
+    } else {
+      // create proofs from same mint
+      proofs = []; // TODO: create proofs from account for amount
+    }
+
+    // TODO: use the proofs to pay the request
+    // 1. get transports from the payment request
+    // 2. send proofs via nostr or POST
+    console.log('proofs', proofs);
+  };
+
+  const confirmationRowData = useMemo(
+    () => [
+      {
+        label: 'Estimated fee',
+        value: data?.fee ? (
+          <MoneyDisplay
+            variant="secondary"
+            money={data.fee}
+            unit={getDefaultUnit(data.fee.currency)}
+          />
+        ) : (
+          <Skeleton className="h-[33px] w-[20px]" />
+        ),
+      },
+      { label: 'From', value: account.name },
+      {
+        label: 'To',
+        value: formatDestination(
+          sendToMintUrl.replace('https://', '').replace('http://', ''),
+        ),
+      },
+      { label: 'Paying', value: formatDestination(paymentRequest.raw) },
+    ],
+    [data, account, sendToMintUrl, paymentRequest],
+  );
+
+  return (
+    <BaseConfirmation amount={amount} onConfirm={handleConfirm}>
+      {confirmationRowData.map((row) => (
+        <ConfirmationRow key={row.label} label={row.label} value={row.value} />
+      ))}
+
+      {decoded.description && (
+        <p className="text-center text-muted-foreground text-sm">
+          {decoded.description}
+        </p>
+      )}
+    </BaseConfirmation>
+  );
+};
+
+const ConfirmationRow = ({
+  label,
+  value,
+}: { label: string; value: string | React.ReactNode }) => {
+  return (
+    <div className="flex items-center justify-between">
+      <p className="text-muted-foreground">{label}</p>
+      <p>{value}</p>
+    </div>
+  );
+};
+
+/**
+ * Base confirmation component that displays the amount, the confirmation rows, and a confirm button
+ */
+const BaseConfirmation = ({
```

Comment:
lets move local helper components before exported ones. that's how we do it in most places already